### PR TITLE
9748 cc page build

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Builds /security/cc page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/cc
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1JLnHR9Xuuc1t6ojrnBuMWir5XABgf1L7WVpGGtlLaBo/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9748

### CODE REVIEW

**Comment by anasereijo** [comment]
Some suggestions/questions:

- [ ] as the URL is `/security/cc` should this page have a sec. nav?

- [ ] on https://ubuntu.com/security/certifications#cis there's a section about CC. I think this should be updated with a link to this new page. @SirSamTumless I left a comment in the [copydoc](https://docs.google.com/document/d/1CiF-dPqG1Y--h4FWfgKB1ft4G3nC2Boa8NT_JG7D6dQ/edit#heading=h.9xddh331a12x) and I tagged you as I don't know who's the best person to ask this.

- [ ] I updated some headings in the doc. please update those

- [ ] CSEC logo: make it width:180px and height:205px

- [ ] table: the heading styles aren't right. please check how it should look like in [vanilla](https://vanillaframework.io/docs/base/tables)

**Comment by anasereijo** [comment]
LGTM

File: templates/security/cc.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <span class="u-sv3"><h1>Common Criteria</h1></span>
+      <span class="u-sv3"><p class="p-heading--four">Deploy regulated and high security workloads on Ubuntu</p></span>
+      <p>Developing and deploying open source workloads on regulated and high security environments can be challenging. There is a plethora of open source software following diverse development methods, and standards, but not always targeting a particular data protection standard. Canonical ensures that <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> provide access to the necessary artifacts to comply with Common Criteria, an international (ISO/IEC 15408) computer security certification.</p>
+      <p>
+        <a href="/support#get-in-touch" class="p-button--positive">Get the Common Criteria certified Ubuntu</a>
```
Comment (bethcollins92) [inline templates/security/cc.html:18]:
```suggestion
        <a href="/support/get-in-touch" class="p-button--positive js-invoke-modal">Get the Common Criteria certified Ubuntu</a>
```

File: templates/security/cc.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      <p>In brief, it is a common methodology to evaluate products' security controls against a set of security claims. The set of security claims is grouped per product and is called a protection profile. There are different protection profiles that apply to different products. The profile Ubuntu derives its security requirements is the Operating System Protection Profile (OSPP).</p>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <h2>Where is Common Criteria accepted?</h2>
+      <p>Internationally a Common Criteria certification is accepted by members of the <a hred="https://www.commoncriteriaportal.org/ccra/members/" class="p-link--external">CCRA</a> agreement and the EU <a href="https://www.sogis.eu/" class="p-link--external">SOGIS</a> members.</p>
```
Comment (bethcollins92) [inline templates/security/cc.html:49]:
```suggestion
      <p>Internationally a Common Criteria certification is accepted by members of the <a href="https://www.commoncriteriaportal.org/ccra/members/" class="p-link--external">CCRA</a> agreement and the EU <a href="https://www.sogis.eu/" class="p-link--external">SOGIS</a> members.</p>
```

File: templates/security/cc.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        <tr>
+          <td>Ubuntu 18.04 LTS</td>
+          <td>x86_64 and IBM Z</td>
+          <td><a class="p-link--external" href="https://www.commoncriteriaportal.org/files/epfiles/Certification%20Report%20-%20Canonical%20Ubuntu%20Server%2018.04%20LTS.pdf">18.04.4</a></td>
+          <td><a class="p-link--external" href="https://security-certs.docs.ubuntu.com/en/cc">Installation instructions</a></td>
+        </tr>
+      </t-body>
+    </table>
+    <p>
+      <a class="p-button--positive" href="/support#get-in-touch">Contact us</a>
```
Comment (bethcollins92) [inline templates/security/cc.html:103]:
```suggestion
      <a class="p-button--positive js-invoke-modal" href="/support/get-in-touch">Contact us</a>
```

File: templates/security/cc.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    </p>
+  </div>
+</section>
+
```
Comment (bethcollins92) [inline templates/security/cc.html:107]: please add the marketo info here the same as the other page

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Looking good, just a few changes inline

File: templates/security/cc.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+      </t-body>
+    </table>
+    <p>
+      <a class="p-button--positive js-invoke-modal" href="/support/get-in-touch">Contact us</a>
+    </p>
+  </div>
+</section>
+
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/esm" data-form-id="3764" data-lp-id="" data-return-url="https://www.ubuntu.com/security/thank-you?product=esm" data-lp-url="">
```
Comment (bethcollins92) [inline templates/security/cc.html:109]: can you copy this from here please?
https://github.com/canonical-web-and-design/ubuntu.com/pull/9862/files

(here you're using the esm page but it's the support one we need. Make sure the data-form-id is the same as the other page too)

Comment (petesfrench) [reply templates/security/cc.html]: @bethcollins92 There we go :+1:

File: templates/security/cc.html [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <span class="u-sv3"><h1>Common Criteria</h1></span>
+      <span class="u-sv3"><p class="p-heading--four">Deploy regulated and high security workloads on Ubuntu</p></span>
+      <p>Developing and deploying open source workloads on regulated and high security environments can be challenging. There is a plethora of open source software following diverse development methods, and standards, but not always targeting a particular data protection standard. Canonical ensures that <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> provide access to the necessary artifacts to comply with Common Criteria, an international (ISO/IEC 15408) computer security certification.</p>
+      <p>
+        <a href="/support#get-in-touch" class="p-button--positive js-invoke-modal">Get the Common Criteria certified Ubuntu</a>
```
Comment (bethcollins92) [inline templates/security/cc.html:18]:
```suggestion
        <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Get the Common Criteria certified Ubuntu</a>
```

**Review by bethcollins92 — APPROVED** [review]
Please commit this one final change then everything LGTM :)

**Pull request merged**